internal/brokers/responses.go

```diff
@@ -0,0 +1,13 @@
+package brokers
+
+const (
+	// AuthAllowed is the response when the authentication is allowed.
+	AuthAllowed = "allowed"
+	// AuthDenied is the response when the authentication is denied.
+	AuthDenied = "denied"
+	// AuthCancelled is the response when the authentication is cancelled.
+	AuthCancelled = "cancelled"
+)
+
+// AuthReplies is the list of all possible authentication replies.
+var AuthReplies = []string{AuthAllowed, AuthDenied, AuthCancelled}
```

Comment:
This list is only used in one place, I would directly add the []string{} there or just make it private (I agree this is easier to have one next to each other)